src/launcher/components/AppItem.jsx

```diff
@@ -62,108 +65,118 @@ const AppItem = ({
-                                </Dropdown.Item>
+                                    {isInstalling ? 'Installing...' : 'Install'}
+                                </Button>
                             )}
-                        </DropdownButton>
-                    </ButtonToolbar>
-                </Col>
-            </Row>
-        </ListGroup.Item>
+                            <DropdownButton
+                                variant={
+                                    installed
+                                        ? 'outline-primary'
+                                        : 'outline-secondary'
+                                }
+                                title=""
+                                alignRight
+                            >
+                                {app.homepage && (
+                                    <Dropdown.Item
+                                        title="Go to app website"
+                                        onClick={onReadMore}
+                                    >
+                                        More info
+                                    </Dropdown.Item>
+                                )}
+                                {app.releaseNote && (
+                                    <Dropdown.Item
+                                        title="Show release notes"
+                                        onClick={() => onShowReleaseNotes(app)}
+                                    >
+                                        Release notes
+                                    </Dropdown.Item>
+                                )}
+                                {installed && (
+                                    <Dropdown.Item
+                                        title="Create a desktop shortcut for this app"
+                                        onClick={onCreateShortcut}
+                                    >
+                                        Create shortcut
+                                    </Dropdown.Item>
+                                )}
+                                {installed && !local && (
+                                    <Dropdown.Item
+                                        title={`Remove ${app.displayName}`}
+                                        disabled={isDisabled}
+                                        onClick={onRemove}
+                                    >
+                                        {isRemoving
+                                            ? 'Uninstalling...'
+                                            : 'Uninstall'}
+                                    </Dropdown.Item>
+                                )}
+                            </DropdownButton>
+                        </ButtonToolbar>
+                    </Col>
+                </Row>
+                {(app?.progress || app?.progress === 0) && (
+                    <ProgressBar now={app.progress} />
+                )}
```

Comment:
Instead of making the special case for zero better check explicitly for `null` and `undefined`:
```suggestion
                {app?.progress == null || <ProgressBar now={app.progress} />}
```